src/v2/Apps/Artwork/Components/TrustSignals/__tests__/BuyerGuarantee.jest.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  BuyerGuaranteeTestQueryRawResponse,
+  BuyerGuaranteeTestQueryResponse,
+} from "v2/__generated__/BuyerGuaranteeTestQuery.graphql"
+import { renderRelayTree } from "v2/DevTools"
+import React from "react"
+import { graphql } from "react-relay"
+import { ExtractProps } from "v2/Utils/ExtractProps"
+import { BuyerGuaranteeFragmentContainer } from "../BuyerGuarantee"
+
+jest.unmock("react-relay")
+
+const render = (
+  artwork: BuyerGuaranteeTestQueryRawResponse["artwork"],
+  extraProps?: Partial<ExtractProps<typeof BuyerGuaranteeFragmentContainer>>
+) =>
+  renderRelayTree({
+    Component: (props: BuyerGuaranteeTestQueryResponse) => (
+      <BuyerGuaranteeFragmentContainer
+        artwork={{
+          ...artwork,
+        }}
+        {...props}
+        {...extraProps}
+      />
+    ),
+    mockData: {
+      artwork,
+    } as BuyerGuaranteeTestQueryRawResponse,
+    query: graphql`
+      query BuyerGuaranteeTestQuery @raw_response_type {
+        artwork(id: "whatevs") {
+          ...BuyerGuarantee_artwork
+        }
+      }
+    `,
+  })
```

Comment:
@sepans - any reason not to use our new [relay test utils](https://github.com/artsy/force/blob/master/src/v2/DevTools/setupTestWrapper.tsx)?

Reply:
Cool. The rest of the `TrustSignal` [components](https://github.com/artsy/force/tree/master/src/v2/Apps/Artwork/Components/TrustSignals/__tests__) had the test this way with fixtures set up and I just followed the pattern. This is one of the places I am not sure to go with consistency or incremental improvement? 🤔

Reply:
Incremental improvement for sure, since the new test helpers remove so much code / boilerplate

Reply:
I converted [all the tests](https://github.com/artsy/force/pull/6675/commits/d122dd9bcdde35b6caee0b0bfd17f4d22cd41999) in that directory to `relay test utils` to improve **and** be consistent. Thanks for the tip @damassi